LICENSE.md

```diff
@@ -16,23 +16,23 @@ Version 2.0.2
 
 **Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
 
-**API**: Software intermediary for interaction with the Mod.
+**API**: Software which is part of the Mod that allows other software to interact with the Mod.
```

Comment:
If that's the case, then the author can't grant rights about the use of CoFHCore. The author doesn't own it. That can't be included as part of the API.